plugins/SitesManager/API.php

```diff
@@ -394,6 +394,22 @@ public function getSitesWithAdminAccess($fetchAliasUrls = false, $pattern = fals
         return $sites;
     }
 
+    /**
+     * Returns the messages to warn users on site deletion.
+     *
+     * @throws Exception if the website ID doesn't exist or the user doesn't have access to it
+     * @param int $idSite
+     * @return string messages to warn users
```

Comment:
```suggestion
     * @return string messages to warn users
     * @internal
     * @unsanitized
```

Such API methods should be marked as internal, so they are not listed in API documentation.
Also for new API methods we should use the `@unsanitized` hint, where possible. This will cause parameters provided for the API not to be sent through `Common::sanitizeInputValue`. As we only expect an int value here, there is no risk at all for using the parameters unsanitized, as integers can't contain manipulated content.

Reply:
Updated :+1: